tests/integration/messagebroker/messagebroker_kafka_test.go

```diff
@@ -314,6 +314,7 @@ func Test_ResetAutoOffsetForConsumer(t *testing.T) {
 	aresp, err := admin.CreateTopic(context.Background(), messagebroker.CreateTopicRequest{
 		Name:          topic,
 		NumPartitions: 1,
+		Config:        map[string]string{"retention.ms": "10000"},
```

Comment:
@sakshi-goyal-razorpay why "retention.bytes" is not added in the config map

Reply:
It is just a test case, so added just a single config. But sure, will add other too